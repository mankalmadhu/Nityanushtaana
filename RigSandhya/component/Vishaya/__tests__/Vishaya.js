import 'react-native';
import React from 'react';
import ವಿಷಯ from '../Vishaya';
import { StyleSheet } from 'react-native';

import Renderer from 'react-test-renderer';

test("ವಿಷಯ" , () => {

expect(Renderer.create(
    <ವಿಷಯ ವಿನ್ಯಾಸ = {StyleSheet.create({})} >
      Some Text
    </ವಿಷಯ>
).toJSON()).toMatchSnapshot();

});