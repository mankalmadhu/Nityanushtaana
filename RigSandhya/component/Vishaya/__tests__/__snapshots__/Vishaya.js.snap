// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ವಿಷಯ 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={Object {}}
>
  Some Text
</Text>
`;
